uwblueprint/humane-society
# F24/alice/onboarding

## Notion ticket link

[Onboarding](https://www.notion.so/uwblueprintexecs/Onboarding-Task-10610f3fb1dc80f5aaf6d22563424aef)



## Implementation description
* Created a new table in the database schema to store information about team members.
* Added a GET API endpoint to retrieve all team members from the database.
* Added a front-end page using the existing framework to display the list of members in a table format.



## Steps to test
1. Run the backend server and verify that the GET and POST APIs are working and information being displayed



## What should reviewers focus on?
* Test for any edge cases 


## Checklist
